### ISSUE TITLE
The javascript syntax highlighting can not recognize blade syntax inside it

### ISSUE THREAD

**Issue by MuathIbnHassan** [inxilpro/IntellijAlpine#7]
thank you for the incredible work
I faced a little problem with syntax highlighting when I did use the plugin on my project which is made using [TALL stack](https://tallstack.dev/)

examble :

```blade
<div x-init="console.log(123,{{data_get($array,'key')}})">

</div>
```

the code inside ```{{ }}``` will not be recognized as PHP

**Comment by inxilpro** [comment]
I've dug into this a little bit and don't have an immediate solution. I'm a PHP dev, so I'm pretty much just figuring things out as I go :)

If anyone wants to submit a PR, I'm definitely open to that!

**Comment by inxilpro** [comment]
This is fixed in 0.4.0!
